pkg/runs/streams.go

```diff
@@ -29,10 +29,10 @@ func ValidateStream(streams []string, stream string) error {
 	}
 
```

Comment:
Code above here returns nil in the middle of a function. Bad practice, will change.